crates/uv-trampoline/src/bounce.rs

```diff
@@ -78,7 +78,17 @@ fn make_child_cmdline() -> CString {
 
     // Only execute the trampoline again if it's a script, otherwise, just invoke Python.
     match kind {
-        TrampolineKind::Python => {}
+        TrampolineKind::Python => {
+            // `std::env::set_var` is safe to call on Windows.
+            unsafe {
```

Comment:
cc @BurntSushi on whether or not we want to document this safety more thoroughly.

Reply:
Ideally yeah. IIRC, this crate is something we vendored from another source (I can't remember where at the moment) that used `unsafe` as a means to keep the binary size very small (by avoiding panicking branches). IDK if that motivation is still relevant or not. If not, perhaps we could remove many uses of `unsafe`.

But yeah ideally we'd follow this blueprint: https://jack.wrenn.fyi/blog/safety-hygiene/